CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 ## Change Log
 
+### v0.8.0
+
+#### Fixed
+
+- Fixed issue where middle names were being used as last names and the last name was completely omitted.
+- Added subnets and security groups for the node-lambda deployment script.
```

Comment:
I forgot to update this last time but I didn't make a deployment for this, so it's bundled in the same version.